tests/python/gaia-ui-tests/gaiatest/apps/search/regions/browser.py

```diff
@@ -24,7 +29,21 @@ class Browser(Base):
     def __init__(self, marionette):
         Base.__init__(self, marionette)
         self.marionette.switch_to_frame()
-        self._root_element = self.marionette.find_element(*self._browser_app_locator)
+        id = self.marionette.execute_script("""
```

Comment:
Nit: use `frame_id` instead of `id`. `id` is already defined in Python and we're covering it up in this line.